CMakeLists.txt

```diff
@@ -74,7 +69,7 @@ else()
 endif()
 
 if (ENABLE_LIBCXX)
-    message(STATUS "Using libc++")
+    message(STATUS "Using libc++ with std::__1::basic_string")
```

Comment:
message(STATUS "Using libc++ with C++11 ABI") 

This message looks better?

Reply:
done